src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const payPerDay = 40;
+  const discountAfterThreeDays = 20;
+  const discountAfterSevenDays = 50;
+
+  if (days < 3) {
+    result = days * payPerDay;
+  } else if (days >= 3 && days < 7) {
+    result = (days * payPerDay) - discountAfterThreeDays;
+  } else {
+    result = (days * payPerDay) - discountAfterSevenDays;
```

Comment:
```suggestion
  }
    result = (days * payPerDay) - discountAfterSevenDays;
```

You can get rid of this `else` here